config/settings/stage.py

```
@@ -0,0 +1,150 @@
+# CELERY_TASK_ALWAYS_EAGER: uses base.py default (False)
+# DOWNLOAD_RETRY_*: uses base.py defaults
+
+# LOGGING
+# ------------------------------------------------------------------------------
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "filters": {"require_debug_false": {"()": "django.utils.log.RequireDebugFalse"}},
+    "formatters": {
+        "verbose": {
+            "format": "%(levelname)s %(asctime)s %(module)s %(process)d %(thread)d %(message)s",  # noqa: E501
```

Ruff (0.14.5)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)